data_tests/user_data.py

```diff
@@ -1,7 +1,7 @@
 user_data_rab_19 = {
     'FIRST_NAME': 'Andrew',
     'LAST_NAME': 'Stuart',
-    'BIRTHDAY': ('07', '04', '1999'),
+    # 'BIRTHDAY': ('07', '04', '1999'),
```

Comment:
why there are comments?

Reply:
Because the test fails for this field. An additional procedure is required to convert date format.

Reply:
I will remove this item at all

Reply:
Done